docker-for-mac/release-notes.md

```diff
@@ -13,6 +13,39 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
+## Docker Desktop 3.6.0
+2021-08-10
+
+[Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn}
+[Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn}
+
+### New
+
+- You can now create a Dev Environment from your local git repository
+- Add sorting to volume list
+- Add name search to volumes list
+
+### Upgrades
+
+- [Compose V2 RC1](https://github.com/docker/compose-cli/releases/tag/v2.0.0-rc.1)
+- Go runtime to 1.16.7
+- Docker Engine to v20.10.8
+- containerd to v1.4.9
+- runc v1.0.1
+- Kubernetes v1.21.3 
+- Golang 1.16.6
```

Comment:
This should have links, I believe. See https://github.com/docker/docker.github.io/blob/f7365935754b11770f6bcf20dab47a6d8c3bcfbd/docker-for-mac/release-notes.md#upgrades-3 for example